### ISSUE TITLE
Informasjonsmodeller - lenke for "i samsvar med" (dct:conformsTo) virker ikke

### ISSUE THREAD

**Issue by ToHelland** [Informasjonsforvaltning/fdk-issue-tracker#375]
Gjelder egenskapen "i samsvar med" (dct:conformsTo) og modeller høstet fra Open API specification. I tillegg vises Versjon uten at den har noen verdi. Hvis det ikke er definert noe versjonsnummer for dct:Standard-klassen", som er range for dct:conformsTo, skal ikke versjonsfeltet vises. 
![image](https://user-images.githubusercontent.com/17720749/123618895-e9df5e00-d808-11eb-9c28-6d12ebdef838.png)

Gjelder modell: https://data.norge.no/informationmodels/14a8758e-d467-3dbf-8a90-64b6914cc45f

**Comment by shenluna** [comment]
Testet Ok, og verifisert med oppgitt informasjonsmodell

**Comment by ToHelland** [comment]
@shenluna @matsjs Her får jeg en 404-feil når jeg klikker på lenken for "Er i samsvar med" (dct:conformsTo).

**Comment by shenluna** [comment]
@ToHelland Det ser ikke ut som https://data.norge.no/specification/kontoopplysninger er en lenke som finnes, eller har innhold. Skulle det vært det?
Ut fra hva jeg kan se viser vi lenker så lenge den er på riktig format, vi sjekker ikke om lenken virker/har innhold.

**Comment by ToHelland** [comment]
@shenluna  Ja, den bør ha innhold. Dette er modeller som høstes på bakgrunn av API'ene, så URI'en for dct:conformsTo er det vi som setter, ikke eierne av API. Derfor må den ha et innhold.

**Comment by shenluna** [comment]
@ToHelland Skal den lenke til APIet? Det ser evt ut som om URI'en som er blitt satt er feil. Jeg flyttet saken tilbake til under arbeid, så kan vi ta en prat med Mats når han er tilbake :)

**Comment by ToHelland** [comment]
Ja, eller selve spesifikasjonen for API'et. Men jeg føler at vi må ta en liten diskusjon der om hva det skal pekes til. For når vi knytter en modell til et API, så bruker vi conformsTo på API'et. Så da får vi at API'et er i samsvar med modellen og modellen er i samsvar med API'et, og fra et filosofisk synspunkt blir dette ikke helt riktig, eller hva @stigbd?

**Comment by matsjs** [comment]
Etter samtale med Stig @stigbd så fant vi at dette skyldes API-beskrivelsen og må i så fall fikses i dataservice-publisher.
Du kan evt flytte issuet Tore? @ToHelland

**Comment by matsjs** [comment]
Etter samtale med Stig @stigbd så fant vi at dette skyldes API-beskrivelsen og må i så fall fikses i dataservice-publisher.
Du kan evt flytte issuet Tore? @ToHelland
